test/controller/pokemon.js

```diff
@@ -1312,6 +1316,21 @@ describe('PokemonController', () => {
         }
       });
     });
+    it('filters clone list based on box visibility', async () => {
```

Comment:
This checks both the clone list filtering them out, and the fact that if we upload to a box that is unlisted, that is done straight away (as mentions in the upper test)